plugin-gestures/src/main/java/com/mapbox/maps/plugin/gestures/GesturesPluginImpl.kt

```diff
@@ -1378,6 +1378,11 @@ class GesturesPluginImpl : GesturesPlugin, GesturesSettingsBase, MapStyleObserve
         return false
       }
 
+      // Skip 2-finger scroll if pinchScrollEnabled is disabled.
+      if (!internalSettings.pinchScrollEnabled && detector.pointersCount > 1) {
```

Comment:
shouldn't we actually check for `detector.pointersCount > 1` instead?

Reply:
@kiryldz we are checking `detector.pointersCount > 1` and `pinchScrollEnabled` here, or did I miss anything?

Reply:
oops, sorry, my bad, nevermind 🤦